packages/test/test-end-to-end-tests/src/testSnapshotCache.ts

```diff
@@ -16,17 +16,20 @@ export interface ValueWithSnapshot {
 }
 
 export class TestSnapshotCache implements IPersistedCache {
-	private readonly cache = new Map<string, ValueWithSnapshot>();
+	private readonly cache = new Map<string, unknown>();
```

Comment:
What's the reason for change of types in the PR?

Reply:
Want to introduce a generic type param `T` on the class that defaults to `unknown`, and use that `T` everywhere?

Reply:
I don't know what can be used here, the issue is that there are snapshots stored here, and something else as well. That's why there was an error in the first place.